README.md

```diff
@@ -1,46 +1,148 @@
 # Open Services Group SCRUM
 
-This repository contains information on how we implement SCRUM and holds issues for
-sprints. Sprints are planned in the [SCRUM BOARD](https://github.com/orgs/open-services-group/projects/21/views/1)
+This repository contains information on how we implement SCRUM and holds issues for sprints.
+Sprints are planned in the [SCRUM BOARD].
 
 # TL;DR;
 
-1. check the stakeholder backlog for assigned issues and prioritize completing them
+1. check the stakeholder backlog for issues assigned to you and well define them, that they can be prioritized
 2. keep your assigned issues in the other columns updated
 3. once done, move it to the `Done` column - don't do it just prior to the meeting
-4. come [prepared](#prior-to-the-meeting) to the sprint meeting
+4. come [prepared] to the sprint meeting
 
 ## Asynchronous work
 
 This work happens all the time and *not* during the sprint planning meeting.
 
 ### Stakeholder Backlog
-
-1. all unassigned issues are reviewed by the product owner
-2. all assigned issues have to be completed by the assignee and unassigned once ready for review
-
-This loop continues until the [product owner](#product-owner) moves it into the product backlog.
+The goal of the [Stakeholder Backlog] is to collect issues and ideas (notes) from all Stakeholders/ users/ contributors/ etc..
+Issues and ideas can be unprioritzed, unsorted and even not well defined.
+The [Product Owner] reviews them regularly and decides, which of these need more attention, need more details and/ or need to be worked on (prioritized).
+
+  1. all issues start in the Stakeholder Backlog
+  1. all new and unassigned issues are reviewed by the [product owner] ([PO])
+  1. the product owner decides if an issue is well defined and needs to be prioritized
+     1. if an issue is well defined, the [PO] sets the label to `triage/accepted`
+     1. if an issue is not well defined, the [PO] assigns it to a contributor e.g. for decscribing it better or get more details
+        1. This loop continues until the [PO] sets the label `triage/accepted` and leaves it unassigned
+     1. if an issue is well defined `triage/accepted` and is prioritized by the [PO], the [PO] moves it to the [Product Backlog]
+  1. all assigned issues have to be completed by the assignee and unassigned once ready for review
```

Comment:
> all assigned issues have to be completed by the assignee and unassigned once ready for review

a bit unclear to me, are you referring to when the PO assigns it to a contributor in  point 3-ii ?

Reply:
yes.
Maybe this needs to be removed (redundant), or updated.
I am not sure, what is the best here yet.
Thanks for bringing it up.